src/tests.utils.ts

```diff
@@ -1,7 +1,5 @@
 import Web3 from "web3";
```

Comment:
Please drop all the changes in this file, mostly all of them are formatting changes, and the `abiDecode` should be in `utils.ts` if needed.